public/javascripts/SVLabel/src/SVLabel/mission/MissionProgress.js

```diff
@@ -49,6 +49,39 @@ function MissionProgress (svl, gameEffectModel, missionModel, modalModel, neighb
         );
         mission.complete();
 
+        // mTurk survey prompt, replace mTurk variable with actual assessment of if modal should display
+        // query if 1. user is mturk 2. user has just completed all missions in their *first* HIT
```

Comment:
user is a turker you mean?

Reply:
This is Steven's comment. I've modified it